core/src/genesis.rs

```diff
@@ -395,9 +405,19 @@ impl RawGenesisDomainBuilder {
 
     /// Add an account to this domain
     #[must_use]
-    pub fn with_account(mut self, account_name: Name, public_key: PublicKey) -> Self {
+    pub fn with_account(self, account_name: Name, public_key: PublicKey) -> Self {
+        self.with_account_meta(account_name, public_key, Default::default())
+    }
+
+    pub fn with_account_meta(
```

Comment:
should we add `#[must_use]` here as well and same about missing docs and Metadata path

Reply:
Better to add `[must_use]` to the type.